lib/brainstem/query_strategies/filter_and_search.rb

```diff
@@ -28,23 +38,25 @@ def run_search(scope, includes)
 
         result_ids, _ = @options[:primary_presenter].run_search(@options[:params][:search], search_options)
         if result_ids
-          result_ids
+          [scope.where(id: result_ids), result_ids]
```

Comment:
Help me understand, what is this for?

Reply:
This just instantiates the scope and returns the ordered array, makes it a little nicer than doing an extra assignment, and I can do `scope, ordered_ids = this_method_call(args_n_stuff)`

Reply:
NON-BLOCKING:

Maybe two line this?

```
resulted_models = scope.where(id: result_ids)
[resulted_models, result_ids]
```

Idk if `resulted_models` is the best name but you get it.

Reply:
Sure man!